app/pods/components/web-components/au-loading-overlay/component.js

```diff
@@ -0,0 +1,20 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+
+export default class LoadingOverlay extends Component {
+  @service() intl;
+
+  get getTitle() {
+    if (this.args.title) {
+      return `${this.args.title}`;
+    }
+    return this.intl.t('loading-text');
+  }
+
+  get getMessage() {
+    if (this.args.message) {
+      return `${this.args.message}`;
+    }
+    return `${this.intl.t('saving-change-message')} ${this.intl.t('please-be-patient')}`;
```

Comment:
Het lijkt me hier misschien nuttiger om gewoon niks terug te geven met een `return null` .. gaan we ervanuit dat de `loadingOverlay` altijd geassocieerd wordt met een save? Anders lijkt het me logischer om niks terug te geven of eerder een generiek bericht.

Reply:
Maar dan toont de modal geen text als je hem oproept? Waarom zou je dat willen?